nginx_stage/share/nginx_stage_example.yml

```diff
@@ -110,6 +110,10 @@
 #
 #pun_error_log_path: '/var/log/ondemand-nginx/%{user}/error.log'
 
+# Custom format configuration for access log
+#
+#pun_log_format: '$server_name $remote_addr - $remote_user [$time_local] "$request" $status $bytes_sent "$http_referer" "$http_user_agent" "$http_x_forwarded_for"'
```

Comment:
Comment here should be updated to match what the default value actually is:

    '$remote_addr - $remote_user [$time_local] "$request" $status $body_bytes_sent "$http_referer" "$http_user_agent" "$http_x_forwarded_for"'

Reply:
Oops! Missed that one. Fixed.